assets/timber.scss.liquid

```diff
@@ -1778,11 +1822,27 @@ label.error {
   margin-bottom: 0;
 }
 
+.site-header__search {
+  display: inline-block;
+  max-width: 400px;
+  width: 100%;
+
+  .site-header__text-links + & {
```

Comment:
Don't think so. This adds some spacing only if `site-header__search` follows a set of text links, which is a conditional module.